### PR TITLE
fix scale combination

### PR DESCRIPTION
**Pull request by ebattat**
This PR fix the following:
1. Scale combination SCALE X SCALE_NODES.
e.g.
SCALE=2
SCALE_NODES= [ 'master-1', 'master-2' ]
It will spin up 2 pods/vms in each node, total 4 vms/pods

2. Add scale stamp for output folder local and s3

### CODE REVIEW

File: benchmark_runner/workloads/workloads_operations.py [hunk 1]
```diff
@@ -77,6 +77,16 @@ def __init__(self):
         if self._enable_prometheus_snapshot:
             self._snapshot = PrometheusSnapshot(oc=self._oc, artifacts_path=self._run_artifacts_path, verbose=True)
 
+    def __get_workload_file_name(self, workload):
+            """
+            This method return workload name
```
Comment (RobertKrawitz) [inline benchmark_runner/workloads/workloads_operations.py:82]: "returns"

**Review by RobertKrawitz — COMMENTED** [review]
One small grammar fix in comment.

File: benchmark_runner/workloads/workloads_operations.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/workloads/workloads_operations.py]: @RobertKrawitz, fix it. please let me know if there are more comments

**Review by RobertKrawitz — APPROVED** [review]
lgtm

**Pull request merged**